src/chains/ethereum/ethereum/src/api.ts

```diff
@@ -1100,15 +1100,18 @@ export default class EthereumApi implements Api {
    * ```javascript
    * // Simple.sol
    * // // SPDX-License-Identifier: MIT
-   * //  pragma solidity ^0.7.4;
+   * //  pragma solidity >= 0.4.22 <0.9.0;
+   * //
+   * // import "console.sol";
    * //
    * //  contract Simple {
    * //      uint256 public value;
    * //      constructor() payable {
+   * //          console.log("Called Simple contract constructor. Setting value to 5.");
```

Comment:
```suggestion
   * //          console.log("Called Simple contract constructor. Setting `value` to 5.");
```
I think this is a little clearer (that "value" refers to the variable name)

Same as per other instances. It's not a significant issue though, as clearly the contracts are fairly simple.

Reply:
¯\_(ツ)_/¯ works for me

Reply:
(I don't have the option to use the suggested change)

Reply:
Seeing your here made my heart hurt.